climakitae/new_core/param_validation/bias_adjust_model_to_station_param_validator.py

```diff
@@ -560,24 +560,21 @@ def _validate_timescale_requirement(query: Dict[str, Any]) -> bool:
 
     # Check if table_id is hourly (1hr or hr)
     if table_id not in ["1hr", "hr"]:
-        import warnings
-
         msg = (
-            f"\n\n"
-            f"╔══════════════════════════════════════════════════════════════════════╗\n"
-            f"║  Station Bias Correction Error: Hourly Data Required                 ║\n"
-            f"╠══════════════════════════════════════════════════════════════════════╣\n"
-            f"║  You requested: table_id='{table_id}'                                  \n"
-            f"║  Required:      table_id='1hr'                                       ║\n"
-            f"║                                                                      ║\n"
-            f"║  Why? HadISD station observations are recorded hourly. Bias         ║\n"
-            f"║  correction can only match hourly model data to hourly observations.║\n"
-            f"║                                                                      ║\n"
-            f"║  Fix: Change .table_id('{table_id}') to .table_id('1hr')               \n"
-            f"╚══════════════════════════════════════════════════════════════════════╝\n"
+            "\\n\\n"
+            "╔══════════════════════════════════════════════════════════════════════╗\\n"
+            "║  Station Bias Correction Error: Hourly Data Required                 ║\\n"
+            "╠══════════════════════════════════════════════════════════════════════╣\\n"
+            "║  You requested: table_id='%s'                                          \\n"
+            "║  Required:      table_id='1hr'                                       ║\\n"
+            "║                                                                      ║\\n"
+            "║  Why? HadISD station observations are recorded hourly. Bias         ║\\n"
+            "║  correction can only match hourly model data to hourly observations.║\\n"
+            "║                                                                      ║\\n"
+            "║  Fix: Change .table_id('%s') to .table_id('1hr')                       \\n"
+            "╚══════════════════════════════════════════════════════════════════════╝\\n"
```

Comment:
Always thinking of things to look at after I hit approve. This formatting looks pretty wonky in ipython or jupyter notebook printout. For example:
<img width="1447" height="232" alt="Screenshot 2025-12-29 at 3 15 58 PM" src="https://github.com/user-attachments/assets/f37d8f11-72fe-4823-b29e-35fe4fa8a8d2" />
Maybe not a big issue, just wanted to point it out.

Reply:
Looks good now!

<img width="1447" height="232" alt="Screenshot 2025-12-30 at 8 27 49 AM" src="https://github.com/user-attachments/assets/b2569b42-31c3-4b61-bc59-a24947e8888f" />